src/github/handlers/issue-comment-created.ts

```diff
@@ -0,0 +1,36 @@
+import { GitHubContext } from "../github-context";
+import { getConfig } from "../utils/config";
+
+export default async function issueCommentCreated(context: GitHubContext<"issue_comment.created">) {
+  const body = context.payload.comment.body.trim();
+  if (/^\/help$/.test(body)) {
+    const comments = [
+      "### Available Commands\n\n",
+      "| Command | Description | Example |",
+      "|---|---|---|",
+      "| `/help` | List all available commands. | `/help` |",
+    ];
+    const configuration = await getConfig(context);
+    for (const pluginArray of Object.values(configuration.plugins)) {
+      for (const plugin of pluginArray) {
+        // Only show plugins that have commands available for the user
+        if (plugin.command) {
+          comments.push(`| \`${getContent(plugin.command)}\` | ${getContent(plugin.description)} | \`${getContent(plugin.example)}\` |`);
+        }
+      }
+    }
+    await context.octokit.issues.createComment({
+      body: comments.join("\n"),
+      issue_number: context.payload.issue.number,
+      owner: context.payload.repository.owner.login,
+      repo: context.payload.repository.name,
+    });
+  }
+}
+
+/**
+ * Ensures that passed content does not break MD display within the table.
+ */
+function getContent(content: string | undefined) {
```

Comment:
```suggestion
function getContent(content?: string) {
```

Reply:
This semantically changes how the function can be called.
With
```TypeScript
function getContent(content: string | undefined)
```
Calling `getContent()` is invalid, because the argument is required.
Instead with
function getContent(content?: string)
Calling `getContent()` would be valid as the argument itself is not required, which doesn't make sense in this situation.

Reply:
Either do my suggestion, or do the following

```ts
const getContent = (content: string) => content.replace("|", "\\|") : "-";
```

It doesn't make sense that you make the parameter optional and then check for it, if you're saying "the argument is required"

Reply:
I see that it must be more convenient to work with in your implementation.

Reply:
That's the thing it is not made optional, it accepts `undefined` or `string` as an input type. But you are required to provide an argument. Calling the function with no arguments should not be an available scenario. That's because eventually the content itself can be undefined (Octokit doesn't set the body to `null` when empty which would have been preferred ofc).